crates/swc_ecma_preset_env/src/corejs3/entry.rs

```diff
@@ -12,33 +12,37 @@ use swc_ecma_visit::{Fold, FoldWith};
 use super::compat::DATA as CORE_JS_COMPAT_DATA;
 
 static ENTRIES: Lazy<AHashMap<String, Vec<&'static str>>> = Lazy::new(|| {
-    serde_json::from_str::<AHashMap<String, Vec<String>>>(include_str!("entries.json"))
-        .expect("failed to parse entries.json from core js 3")
-        .into_iter()
-        .map(|(k, v)| {
-            (
-                k,
-                v.into_iter()
-                    .map(|s: String| &*Box::leak(s.into_boxed_str()))
-                    .collect::<Vec<_>>(),
-            )
-        })
-        .collect()
+    serde_json::from_str::<AHashMap<String, Vec<String>>>(include_str!(
+        "../../../../node_modules/core-js-compat/entries.json"
```

Comment:
`error: couldn't read /Users/longyinan/.cargo/registry/src/github.com-1ecc6299db9ec823/swc_ecma_preset_env-0.105.0/src/corejs3/../../../../node_modules/core-js-compat/modules-by-versions.json: No such file or directory (os error 2)`

Reply:
https://github.com/swc-project/swc/issues/3971 is the issue for this